scripts/e2e-test.sh

```diff
@@ -44,6 +44,8 @@ helm template deploy/helm/eunomia-operator/ \
 
```

Comment:
Can we just add the code from shell function `check_image()` here? Then I think could get rid of the `cripts/test-eunomia.sh` script.

Reply:
@seanmalloy Removed ` test-eunomia.sh` added pod status check in `e2e-test.sh` itself .

Reply:
Yes, I believe that will work.